modules/nw-udn-cr.adoc

```diff
@@ -16,6 +16,27 @@ The following procedure creates a user-defined network that is namespace scoped.
 
 .Procedure
 
+. Optional: For a `UserDefinedNetwork` CR that uses a primary network, create a namespace with the `k8s.ovn.org/primary-user-defined-network` label by entering the following command:
++
+[source,yaml]
+----
+$ cat << EOF | oc apply -f -
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: <udn_namespace_name>      
+  labels:
+    k8s.ovn.org/primary-user-defined-network: ""
+EOF
+----
+
+. Optional: If the user-defined network is intended to be used by a non-administrator user, grant the non-administrator user `admin` permissions on the namespace by entering the following command:
```

Comment:
Please see https://github.com/openshift/openshift-docs/pull/86845#discussion_r1954723749

Reply:
Removed ". Optional: If the user-defined network is intended to be used by a non-administrator user, grant the non-administrator user `admin` permissions on the namespace by entering the following command:"

Reply:
Thanks @stevsmit @ormergi for catching this. I believe that was the main change in this doc. rest looks good to me.  LGTM